classes/identity/Identity.inc.php

```diff
@@ -116,14 +116,14 @@ function setGivenName($givenName, $locale) {
 	 * Get the localized given name
 	 * @return string
 	 */
-	function getLocalizedGivenName() {
-		return $this->getLocalizedData(IDENTITY_SETTING_GIVENNAME);
+	function getLocalizedGivenName($defaultLocale = null) {
+		return $this->getLocalizedData(IDENTITY_SETTING_GIVENNAME, $defaultLocale);
 	}
 
 	/**
 	 * Get family name.
 	 * @param $locale string
-	 * @return string
+	 * @return string|array
```

Comment:
This will always return an `array`, won't it?

Reply:
@asmecher, not if you provide a `locale` other than `null`, I think.

Reply:
Ah, of course :)